network-area-diagram/src/main/java/com/powsybl/nad/svg/SvgWriter.java

```diff
@@ -129,13 +129,57 @@ private void drawBranchEdges(Graph graph, XMLStreamWriter writer) throws XMLStre
 
             drawHalfEdge(graph, writer, edge, BranchEdge.Side.ONE);
             drawHalfEdge(graph, writer, edge, BranchEdge.Side.TWO);
+
+            if (BranchEdge.PST_EDGE.equals(edge.getType())) {
+                drawPstArrow(writer, edge);
+            }
+            if (edge.getType().equals(BranchEdge.HVDC_LINE_EDGE)) {
+                drawConverterStation(writer, edge);
+            }
+            if (BranchEdge.LINE_EDGE.equals(edge.getType()) && svgParameters.isLineNameDisplayed()) {
```

Comment:
Did you try to display the name also above two-winding transformers, phase-shift transformers and converter stations?

Reply:
Displaying names above phase-shift transformers, converter stations and two-winding transformers does not look good because of the drawing already present at the center of the edge. I was thinking of writing the names slightly above the edge to avoid this but when there are several lines carrying two-winding transformers for example running in parallel, the problem remains. It looks a bit messy.

Reply:
As discussed this morning, the diagram remains readable with names on every type of edge. Since this piece of information seems meaningful for users, we finally choose to display names, regardless of the edge type. The HVDC converter station design is slightly modified to be more visible even with the edge name displayed.